File: BEE 1118.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var newNum;
while(true){
    var sum = 0;
    var cnt = 0;
    while(cnt != 2){
        var num = parseFloat(lines.shift());
        if (num >= 0 && num <= 10){
            sum += num;
            cnt += 1;
        }
        else{
            console.log("nota invalida");
        }
    }
    console.log("media =", (sum / cnt).toFixed(2));
    while(true){
        console.log("novo calculo (1-sim 2-nao)");
        newNum = parseInt(lines.shift());
        if (newNum < 1 || newNum > 2){
            continue;
        }
        else{
            break;
        }
    }
    if (newNum == 2){
        break;
    }
}

File: BEE 1118.py
while(True):
    sum = 0
    cnt = 0

    while(cnt != 2):
        num = float(input())
        if (num >= 0 and num <= 10):
            sum += num
            cnt += 1
        else:
            print("nota invalida")

    print("media = {:.2f}".format(sum / cnt))
    newNum = 0
    while(True):
        print("novo calculo (1-sim 2-nao)")
        newNum = int(input())
        if (newNum == 1 or newNum == 2):
            break

    if (newNum == 2):
        break